dataModel.NeMoBil/Trip/schema.json

```diff
@@ -0,0 +1,578 @@
+{
+  "required" : [ "id", "type" ],
+  "type" : "object",
+  "properties" : {
+    "id" : {
+      "type" : "string",
+      "description" : "Property. Unique identifier of the trip"
+    },
+    "type" : {
+      "type" : "string",
+      "description" : "Property. NGSI Entity type. It has to be Trip",
+      "enum" : [ "Trip" ]
+    },
+    "user" : {
+      "type" : "string",
+      "description" : "Relationship. User that booked the trip"
+    },
+    "vehicle" : {
+      "type" : "string",
+      "description" : "Relationship. Vehicle used to fulfill the trip"
+    },
+    "pickupLocation" : {
```

Comment:
instead of including the detail you can insert this ref clause 
"picupLocation": {
 "$ref": "https://smart-data-models.github.io/data-models/common-schema.json#/definitions/Location-Commons",
 "description" : "GeoProperty. Geojson reference to the item. It can be Point, LineString, Polygon, MultiPoint, MultiLineString or MultiPolygon. Should be a reference to location"}